web/src/components/navbar/NavBar.test.tsx

```diff
@@ -102,6 +102,7 @@ describe("<NavBar />", () => {
   beforeEach(() => {
     jest.resetAllMocks();
     useMockRouter({});
+    useMockUserData({});
```

Comment:
dust - I'm confused about why this is needed? Given how no other modifications were made and I assume things were passing here before hand?

Reply:
It's because the sub component now uses userData to get the businessName